packages/shared/src/contexts/AuthContext.tsx

```diff
@@ -79,6 +80,7 @@ export const AuthContextProvider = ({
   const authContext: AuthContextData = useMemo(
     () => ({
       user: user && 'providers' in user ? user : null,
+      isFirstVisit: true,
```

Comment:
Can't remember exactly, but, won't this value be coming from boot, right?

Reply:
Oh yes, wow great catch!
I overwritten this locally 🤯 (Should have been reverted back)